src/main/scss/templates.base/_li-nav-page.scss

```diff
@@ -6,8 +6,7 @@
 	display: flex;
```

Comment:
Not sure where to put this (since I believe it's from an earlier commit), but can the nav-bar have a higher z-index? I thought material design would handle the layering (?). When scrolling, I noticed this:

![image](https://cloud.githubusercontent.com/assets/13812345/26492486/3db5f10e-41c9-11e7-9947-5aa1e0703f4a.png)